clients/drcachesim/tests/missfile-config-file.templatex

```diff
@@ -7,30 +7,22 @@ Core #0 \([0-9] traced CPU\(s\): #.*\)
     Warmup misses:                *[0-9,\.]*..
     Hits:                         *[0-9,\.]*....
     Misses:                       *[0-9,\.]*..
-    Invalidations:                *[0-9,\.]*..
-.*    Miss rate:                        *[0-9][,\.]..%
+    Invalidations:                *[0-9,\.]*[\s\S]*
```

Comment:
suggest: This is equivalent to `.*`, right?  Which is already there, for skipping prefetch lines.  So maybe more consistent to leave the `.*` and just remove the `Miss rate` so the `.*` is next to `L1D`?

Reply:
Done.